modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/index.js

```diff
@@ -1,17 +1,38 @@
 import {useContext} from 'react';
 import {AppContext} from '../context';
+import ActivationKeys from './ActivationKeys';
 import Home from './Home';
 import Overview from './Overview';
 
 const Pages = () => {
-	const [{page, userAccount}] = useContext(AppContext);
+	const [{page, project, userAccount}] = useContext(AppContext);
 
 	if (page === 'overview') {
```

Comment:
Can we create a constant for all available pages? to avoid bad spelling

```suggestion
	if (page === pages.OVERVIEW) {
```